src/crud.js

```diff
@@ -0,0 +1,67 @@
+const taskDescription = document.querySelector('#txt-input');
+const deleteAllBtn = document.querySelector('.clear-all');
+
+const getFromLocalStorage = () => {
+  const storage = JSON.parse(localStorage.getItem('lstore')) || [];
+  return storage;
+};
+
+const tasks = getFromLocalStorage();
+
+const addToLocalStorage = () => {
+  const storage = JSON.stringify(tasks);
+  localStorage.setItem('lstore', storage);
+};
+
+const addToTasks = (e) => {
+  e.preventDefault();
+  tasks.push({
+    description: taskDescription.value,
+    completed: false,
+    index: tasks.length,
+  });
+  addToLocalStorage();
+  window.location.reload();
```

Comment:
Reloading the browser after every addition or deletion of todo is not a good user experience. I think you should have used target parentNode to grab onto the ui elements to make it easier to make the interface reactive.